charts/fees-register-api/values.yaml

```diff
@@ -1,6 +1,7 @@
 java:
   image: 'hmctspublic.azurecr.io/fees-register/api:latest'
   applicationPort: 8080
+  readinessPath: '/health/readiness'
```

Comment:
you shouldn't need to override this, the new chart has this as default

Reply:
@timja Necessary changes have been done